virttest/utils_misc.py

```diff
@@ -1703,6 +1705,30 @@ def read_from_node_meminfo(self, node_id, key):
         """
         return self.get_all_node_meminfo()[node_id][key]
 
+    def get_all_node_cpus(self):
```

Comment:
rename to get_online_cpus ?

Reply:
> rename to get_online_cpus ?

Yes, you are right, it is about `online` cpus,  and it is a node and cpu mapping, i want to "node". 
And keep consistent with `get_all_node_meminfo`, i prefer the current name more ~   how about keep it ?